R/load.project.R

```diff
@@ -211,6 +211,15 @@ load.project <- function(override.config = NULL)
                      filename,
                      variable.name))
 
+        ## Most times, variable.name will correspond to one loaded object
+        ## but in the case of xls(x) readers, they always return
+        ## variable.name.Sheet1 ... variable.name.SheetN
+        if (grepl('xls', extension)) {
+          variable.name <- grep(paste(variable.name, 'Sheet', sep ='.'),
+                                ls(.TargetEnv),
+                                value = TRUE)
```

Comment:
`data.files.loaded` is returned and loaded into `my.project.info$data`. `my.project.info$data` is then used in `.convert.to.data.table` to find loaded data frames and convert to data tables. Would it be possible to put the logic in `.convert.to.data.table`? Could we check for *any* data frame  that begins with the loaded variable name. This would catch `myvar.sheet1` as well as `myvar.myworksheet` and any other data frames created with a reader returning multiple datasets. It also fences in this change to only projects converting to data tables -- those projects that would most benefit from this change.

Comments and thoughts?